lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -350,6 +350,8 @@ CRI_KBV_J2:
       checkIfDisabled:
         f2f:
           targetState: PYI_CRI_ESCAPE_NO_F2F
+    next:
```

Comment:
Not sure if this was needed as you're about to make the state redundant

Reply:
In the second PR, we will remove this, but I added it to ensure that the two pages are the same.